pkg/remotewrite/config.go

```diff
@@ -1,108 +1,85 @@
 package remotewrite
 
 import (
+	"crypto/tls"
 	"encoding/json"
-	"net/url"
+	"net/http"
 	"strconv"
 	"strings"
 	"time"
 
+	"github.com/grafana/xk6-output-prometheus-remote/pkg/remote"
 	"github.com/kubernetes/helm/pkg/strvals"
-	promConfig "github.com/prometheus/common/config"
-	"github.com/prometheus/common/model"
-	"github.com/prometheus/prometheus/storage/remote"
+
 	"go.k6.io/k6/lib/types"
 	"gopkg.in/guregu/null.v3"
 )
 
 const (
-	defaultPrometheusTimeout = time.Minute
-	defaultFlushPeriod       = 5 * time.Second
-	defaultMetricPrefix      = "k6_"
+	defaultURL          = "http://localhost:9090/api/v1/write"
+	defaultTimeout      = 5 * time.Second
+	defaultPushInterval = 5 * time.Second
+	defaultMetricPrefix = "k6_"
```

Comment:
From this I am left thinking you can configure both the metricPrefix and the timeout, but you can't. Maybe open issues about it?

Reply:
The question now is, do we want them? Because if we think they are required then I can just fix them in https://github.com/grafana/xk6-output-prometheus-remote/pull/48.

Instead, if we think we should wait for users then I will open an issue and add a comment for it in the code. (I would prefer this option for keeping the extension as light as possible).

Reply:
I think opening issues for now is sufficient - we can always add them later

Reply:
#52 #51